dashboard/app/controllers/application_controller.rb

```diff
@@ -28,7 +28,8 @@ class ApplicationController < ActionController::Base
 
   before_action :clear_sign_up_session_vars
 
-  before_action :initialize_statsig_stable_id
+  helper_method :statsig_stable_id
```

Comment:
Do I need this here?

Reply:
Answer: yes, this is to make this method (defined in the controller) available to call in the view